app/api/register/route.ts

```diff
@@ -21,16 +45,47 @@ export async function POST(request: Request) {
             return NextResponse.json({ error: "Not a UCI affiliated email" }, { status: 400 });
         }
 
+        // Check password (no empty password, only unicode chars, no whitespace)
+        const isUnicode = /[^\u0000-\u00ff]/;
```

Comment:
```suggestion
        const isUnicode = /[^\u0021-\u007e]/;
```
only ascii characters